skymp5-functions-lib/src/utils/parseChatMessage.ts

```diff
@@ -67,22 +67,34 @@ export const parseMessage = (text: string): ChatText[] => {
         lastIndex = i;
         currentType.pop();
       } else {
-        if (map[char].double && char !== text[i + 1]) {
-          continue;
-        } else {
-          i += 1;
+        if (map[char].double) {
+          if (char !== text[i + 1]) {
+            continue;
+          } else {
+            i += 1;
+          }
         }
-        if (map[char].isSeparate && (stack.length !== 0 || currentType.length !== 0)) {
+        if (map[char].canBeNested === false && (stack.length !== 0 || currentType.length !== 0)) {
```

Comment:
`!canBeNested`?

Reply:
This won't work, because right now it checks when it's strictly false. If we use `!canBeNested` it will also be true when it's `undefined`.

Reply:
Hm I thought `canBeNested` is a required field. I think booleans with `true` as default are a bit counterintuitive. But `cantBeNested` seems to also be counterintuitive. Maybe `unnestable`, but also sounds strange